### PR TITLE
chore(Link): deprecate enums

### PR DESCRIPTION
**Pull request by rivka-ungar**
https://monday.monday.com/boards/3532714909/views/80492480/pulses/7021050590

### CODE REVIEW

File: packages/core/src/components/Link/Link.types.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export type LinkTarget = "_blank" | "_self" | "_parent" | "_top";
+
+export type IconPosition = "start" | "end";
```
Comment (talkor) [inline packages/core/src/components/Link/Link.types.ts:3]: I think we already have this, maybe in a different PR, we need to make sure we're not exporting it twice, I assume build will fail

File: packages/core/src/storybook/stand-alone-documentaion/typography/typography-migration-guide.mdx [hunk 2]
```diff
@@ -67,7 +67,7 @@ Each of these components can get the variants as a prop. You also get out-of-the
 </Heading>
 ```
 
-For a more detailed breakdown of the variants see the <Link href="/?path=/docs/foundations-typography--docs#type-styles" withoutSpacing target={Link.targets.PARENT}>Type Styles table</Link>.
+For a more detailed breakdown of the variants see the <Link href="/?path=/docs/foundations-typography--docs#type-styles" withoutSpacing target="parent">Type Styles table</Link>.
```
Comment (talkor) [inline packages/core/src/storybook/stand-alone-documentaion/typography/typography-migration-guide.mdx:70]: Shouldn't be `_parent`?

File: packages/core/src/storybook/stand-alone-documentaion/welcome/contributor/contributor.jsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ export const Contributor = ({ imgSrc, className, fullName, title, email }) => {
     <Flex justify={Flex.justify.SPACE_BETWEEN}>
       <InformationBoxTitle>{fullName}</InformationBoxTitle>
       {email && (
-        <Link tabIndex="-1" href={`mailto:${email}`} target={Link.targets.PARENT}>
+        <Link tabIndex="-1" href={`mailto:${email}`} target="parent">
```
Comment (talkor) [inline packages/core/src/storybook/stand-alone-documentaion/welcome/contributor/contributor.jsx:16]: Shouldn't be `_parent`?

File: packages/core/src/storybook/stand-alone-documentaion/typography/typography-migration-guide.mdx (hunk 2 above)
Comment (rivka-ungar) [reply packages/core/src/storybook/stand-alone-documentaion/typography/typography-migration-guide.mdx]: yes, fixed!

File: packages/core/src/components/Link/Link.types.ts (hunk 1 above)
Comment (rivka-ungar) [reply packages/core/src/components/Link/Link.types.ts:3]: Are you referring to this?
https://github.com/mondaycom/vibe/pull/2263/files#diff-b41595a7dcb49d9e0d4849662a262434ccdc9423cfc8b24d3e8ae14f49803cf9R1

**Review by talkor — APPROVED** [review]

**Pull request merged**